### ISSUE TITLE
NER任务效果不佳

### ISSUE THREAD

**Issue by arbalest339** [PaddlePaddle/ERNIE#294]
我在复现时对NER任务在原数据原参数上尝试跑了4次，每一次模型在验证集和测试集上的表现都很差，准确率，召回率，f1，均只有0.5左右。阅读问题列表后我发现我不是个例，也有人在其它任务上也遇到了模型失效的情况，现在尝试的分类任务也是这个问题，已经是最后一个epoch，正确率还是只有0.5。现在环境上唯一的问题可能是我的cudnn版本只有7.2，CUDA9.0，nccl是最新的，这些都没问题。下面贴出最近一次NER任务的运行截图，不知道究竟是怎么回事。
![image](https://user-images.githubusercontent.com/46928336/63644509-131a9900-c71d-11e9-98d3-57a7a52354b0.png)

**Comment by zhanghan1992** [comment]
1.检查下数据格式；2.检查下是否正确加载预训练模型；3.调参数(学习率、batch_size)

**Comment by zhanghan1992** [comment]
看你的情况像是没正确加载预训练模型。贴个完整的log吧

**Comment by arbalest339** [comment]
@zhanghan1992 以下是训练开始之前的所有输出：
+ export FLAGS_eager_delete_tensor_gb=0
+ export FLAGS_sync_nccl_allreduce=1
+ export CUDA_VISIBLE_DEVICES=1
+ python -u run_sequence_labeling.py --use_cuda true --do_train true --do_val true --do_test true --batch_size 16 --init_pretraining_params /home/lzk/ERNIE-devlop/model/params --num_labels 7 --chunk_scheme IOB --label_map_config /home/lzk/ERNIE-devlop/data/msra_ner/label_map.json --train_set /home/lzk/ERNIE-devlop/data/msra_ner/train.tsv --dev_set /home/lzk/ERNIE-devlop/data/msra_ner/dev.tsv --test_set /home/lzk/ERNIE-devlop/data/msra_ner/test.tsv --vocab_path /home/lzk/ERNIE-devlop/model/vocab.txt --ernie_config_path /home/lzk/ERNIE-devlop/model/ernie_config.json --checkpoints ./checkpoints --save_steps 100000 --weight_decay 0.01 --warmup_proportion 0.0 --validation_steps 100 --epoch 6 --max_seq_len 256 --learning_rate 5e-5 --skip_steps 10 --num_iteration_per_drop_scope 1 --random_seed 1
-----------  Configuration Arguments -----------
batch_size: 16
checkpoints: ./checkpoints
chunk_scheme: IOB
dev_set: /home/lzk/ERNIE-devlop/data/msra_ner/dev.tsv
diagnostic: None
diagnostic_save: None
do_lower_case: True
do_test: True
do_train: True
do_val: True
doc_stride: 128
enable_ce: False
epoch: 6
ernie_config_path: /home/lzk/ERNIE-devlop/model/ernie_config.json
for_cn: True
in_tokens: False
init_checkpoint: None
init_pretraining_params: /home/lzk/ERNIE-devlop/model/params
is_classify: True
is_regression: False
label_map_config: /home/lzk/ERNIE-devlop/data/msra_ner/label_map.json
learning_rate: 5e-05
loss_scaling: 1.0
lr_scheduler: linear_warmup_decay
max_answer_length: 100
max_query_length: 64
max_seq_len: 256
metric: simple_accuracy
metrics: True
n_best_size: 20
num_iteration_per_drop_scope: 1
num_labels: 7
predict_batch_size: None
random_seed: 1
save_steps: 100000
shuffle: True
skip_steps: 10
task_id: 0
test_save: test_result
test_set: /home/lzk/ERNIE-devlop/data/msra_ner/test.tsv
tokenizer: FullTokenizer
train_set: /home/lzk/ERNIE-devlop/data/msra_ner/train.tsv
use_cuda: True
use_fast_executor: False
use_fp16: False
use_multi_gpu_test: False
validation_steps: 100
verbose: False
vocab_path: /home/lzk/ERNIE-devlop/model/vocab.txt
warmup_proportion: 0.0
weight_decay: 0.01
------------------------------------------------
attention_probs_dropout_prob: 0.1
hidden_act: gelu
hidden_dropout_prob: 0.1
hidden_size: 768
initializer_range: 0.02
max_position_embeddings: 512
num_attention_heads: 12
num_hidden_layers: 12
sent_type_vocab_size: 4
task_type_vocab_size: 16
vocab_size: 30522
------------------------------------------------
Device count: 1
Num train examples: 20864
Max train steps: 7824
Num warmup steps: 0
W0825 17:24:58.339669 15200 device_context.cc:261] Please NOTE: device: 0, CUDA Capability: 61, Driver API Version: 9.0, Runtime API Version: 9.0
W0825 17:24:58.342540 15200 device_context.cc:269] device: 0, cuDNN Version: 7.0.
W0825 17:24:58.342555 15200 device_context.cc:293] WARNING: device: 0. The installed Paddle is compiled with CUDNN 7.3, but CUDNN version in your machine is 7.2, which may cause serious incompatible bug. Please recompile or reinstall Paddle with compatible CUDNN version.
Load pretraining parameters from /home/lzk/ERNIE-devlop/model/params.
ParallelExecutor is deprecated. Please use CompiledProgram and Executor. CompiledProgram is a central place for optimization and Executor is the unified executor. Example can be found in compiler.py.
W0825 17:24:58.639469 15200 graph.h:204] WARN: After a series of passes, the current graph can be quite different from OriginProgram. So, please avoid using the `OriginProgram()` method!
2019-08-25 17:24:58,642-WARNING: 
     You can try our memory optimize feature to save your memory usage:
         # create a build_strategy variable to set memory optimize option
         build_strategy = compiler.BuildStrategy()
         build_strategy.enable_inplace = True
         build_strategy.memory_optimize = True
         
         # pass the build_strategy to with_data_parallel API
         compiled_prog = compiler.CompiledProgram(main).with_data_parallel(
             loss_name=loss.name, build_strategy=build_strategy)
      
     !!! Memory optimize is our experimental feature !!!
         some variables may be removed/reused internal to save memory usage, 
         in order to fetch the right value of the fetch_list, please set the 
         persistable property to true for each variable in fetch_list

         # Sample
         conv1 = fluid.layers.conv2d(data, 4, 5, 1, act=None) 
         # if you need to fetch conv1, then:
         conv1.persistable = True

                 
I0825 17:24:58.868110 15200 build_strategy.cc:285] SeqOnlyAllReduceOps:0, num_trainers:1
ParallelExecutor is deprecated. Please use CompiledProgram and Executor. CompiledProgram is a central place for optimization and Executor is the unified executor. Example can be found in compiler.py.
W0825 17:24:58.967775 15200 graph.h:204] WARN: After a series of passes, the current graph can be quite different from OriginProgram. So, please avoid using the `OriginProgram()` method!
2019-08-25 17:24:58,968-WARNING: 
     You can try our memory optimize feature to save your memory usage:
         # create a build_strategy variable to set memory optimize option
         build_strategy = compiler.BuildStrategy()
         build_strategy.enable_inplace = True
         build_strategy.memory_optimize = True
         
         # pass the build_strategy to with_data_parallel API
         compiled_prog = compiler.CompiledProgram(main).with_data_parallel(
             loss_name=loss.name, build_strategy=build_strategy)
      
     !!! Memory optimize is our experimental feature !!!
         some variables may be removed/reused internal to save memory usage, 
         in order to fetch the right value of the fetch_list, please set the 
         persistable property to true for each variable in fetch_list

         # Sample
         conv1 = fluid.layers.conv2d(data, 4, 5, 1, act=None) 
         # if you need to fetch conv1, then:
         conv1.persistable = True

                 
share_vars_from is set, scope is ignored.
I0825 17:24:59.020234 15200 build_strategy.cc:285] SeqOnlyAllReduceOps:0, num_trainers:1

没有报模型的错误，预训练模型的目录也是对的
![image](https://user-images.githubusercontent.com/46928336/63648161-31ed4f80-c75e-11e9-95b2-50173b4ab4f5.png)

**Comment by zhanghan1992** [comment]
你在用英文的模型/配置跑中文的任务。先下载中文的模型/配置吧。

**Comment by arbalest339** [comment]
原来如此，非常感谢
